### PR TITLE
send email sample add

### PR DESCRIPTION
**Pull request by moirf**
## Purpose

* send email sample add

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe: send email sample addition
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: send-email/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- azure-communication-services
+---
+
+
+# Email Sample
+
+## Overview
+
+This is a sample email application to show how we can use the `@azure/communication-email` package to build an email experience.
+This sample sends an email to the required recipients of any domain using [Email Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/email/create-email-communication-resource).
```
Comment (natekimball-msft) [inline send-email/README.md:17]: "...required recipients of any domain using **an** [Email Communication Services resource]..."

File: send-email/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+The application is a console based application build using Node.js 14.
```
Comment (natekimball-msft) [inline send-email/README.md:18]: "console-based application" should have a hyphen, or just rephrase to "This is a console application build (or _built_) using..."

File: send-email/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
+2. `git clone https://github.com/Azure-Samples/communication-services-javascript-quickstarts.git`
+
+### Locally configuring the application
+
+3. Open the send-email.js file to configure the following settings
+  - `connectionstring`: Replace `<ACS_CONNECTION_STRING>` with Azure Communication Service resource's connection string.
+  - `sender`: Replace `<SENDER_EMAIL>` with sender email obtained from Azure Communication Service.
+  - `Line 13 - <RECIPIENT_EMAIL>`: Replace with recipient email.
```
Comment (matthohn-msft) [inline send-email/README.md:49]: Format this line like the others in this list.
 `recipient`: Replace <RECIPIENT_EMAIL> with recipient email.

File: send-email/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## Code structure
+
+- ./send-email/package.json: Package file
+- ./send-email/send-email.js: Entry point into the email sample
+
+## Before running the sample for the first time
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
```
Comment (natekimball-msft) [inline send-email/README.md:41]: "...Command Prompt, or an equivalent program, and..."

File: send-email/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+2. `git clone https://github.com/Azure-Samples/communication-services-javascript-quickstarts.git`
+
+### Locally configuring the application
+
+2. Open the send-email.js file to configure the following settings
+  - `connectionstring`: Replace `<ACS_CONNECTION_STRING>` with Azure Communication Service resource's connection string.
```
Comment (natekimball-msft) [inline send-email/README.md:47]: False possessive. Rephrase to "Replace `<ACS_CONNECTION_STRING>` with the connection string found within the Azure Communication Service resource."

File: send-email/README.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  - `sender`: Replace `<SENDER_EMAIL>` with sender email obtained from Azure Communication Service.
```
Comment (natekimball-msft) [inline send-email/README.md:48]: "Replace `<SENDER_EMAIL>` with **the** sender email..."

File: send-email/README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+  - `Line 13 - <RECIPIENT_EMAIL>`: Replace with recipient email.
```
Comment (natekimball-msft) [inline send-email/README.md:49]: "Replace with **the** recipient email."

File: send-email/README.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  - `emailContent`: Either use PlainText or Html to set the email content.
+
+### Local run
+
+    - update the connection string, sender email, recipient's email.
```
Comment (natekimball-msft) [inline send-email/README.md]: - Update the connection string, sender email, and recipient email.

File: send-email/README.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+This is a sample email application to show how we can use the `@azure/communication-email` package to build an email experience.
+This sample sends an email to the required recipients of any domain using [Email Communication Services resource](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/email/create-email-communication-resource).
+The application is a console based application build using Node.js 14.
+
+Additional documentation for this sample can be found on [Microsoft Docs](https://docs.microsoft.com/en-us/azure/communication-services/concepts/email/email-overview).
+
+
+## Prerequisites
+
+- [Visual Studio Code (Stable Build)](https://code.visualstudio.com/download)
```
Comment (natekimball-msft) [inline send-email/README.md:25]: nit: period at end of sentence.

File: send-email/README.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+- [Node.js (~14)](https://nodejs.org/download/release/v14.19.1/)
```
Comment (natekimball-msft) [inline send-email/README.md:26]: nit: period at end of sentence.

File: send-email/README.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+- ./send-email/send-email.js: Entry point into the email sample
+
+## Before running the sample for the first time
+
+1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
+2. `git clone https://github.com/Azure-Samples/communication-services-javascript-quickstarts.git`
+
+### Locally configuring the application
+
+2. Open the send-email.js file to configure the following settings
```
Comment (natekimball-msft) [inline send-email/README.md]: nit: colon at end of this line.

File: send-email/README.md [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+  - `connectionstring`: Replace `<ACS_CONNECTION_STRING>` with Azure Communication Service resource's connection string.
+  - `sender`: Replace `<SENDER_EMAIL>` with sender email obtained from Azure Communication Service.
+  - `Line 13 - <RECIPIENT_EMAIL>`: Replace with recipient email.
+  - `emailContent`: Either use PlainText or Html to set the email content.
+
+### Local run
+
+    - update the connection string, sender email, recipient's email.
+    - `npm install` from the root directory
```
Comment (natekimball-msft) [inline send-email/README.md:54]: nit: period at end of sentence.

File: send-email/README.md [hunk 13]
```diff
@@ -0,0 +1,60 @@
...
+    - `npm start` from the root directory
```
Comment (natekimball-msft) [inline send-email/README.md:55]: nit: period at end of sentence.

File: send-email/README.md [hunk 14]
```diff
@@ -0,0 +1,60 @@
...
+
+
+## ❤️ Feedback
+We appreciate your feedback and energy helping us improve our services. [Please let us know if you are satisfied with ACS through this survey](https://microsoft.qualtrics.com/jfe/form/SV_5dtYL81xwHnUVue).
```
Comment (natekimball-msft) [inline send-email/README.md:59]: We appreciate your feedback and energy **in** helping us improve our services.

**Review by natekimball-msft — CHANGES_REQUESTED** [review]
Just added some minor grammatical considerations. Great work!

**Review by natekimball-msft — APPROVED** [review]

**Review by kagbakpem — APPROVED** [review]

**Pull request merged**